code/datums/manufacturing.dm

```diff
@@ -254,6 +254,14 @@
 	time = 5 SECONDS
 	category = "Tool"
 
+/datum/manufacture/crowbar/purple
+	name = "Crowbar"
+	item_requirements = list("metal" = 1)
+	item_outputs = list(/obj/item/crowbar/purple)
+	create = 1
+	time = 5 SECONDS
+	category = "Tool"
+
```

Comment:
If this is unused it shouldn't be added, afaik no research job starts with a crowbar?

Reply:
This item would be unobtainable, removed the manufacture datum though